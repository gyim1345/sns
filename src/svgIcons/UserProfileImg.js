import React from 'react';
import { css } from '@emotion/core';

function UserProfileImg({ userImage }) {
  return <img css={[imgCss]} src={userImage} alt="" width={28} height={28} />;
}

const imgCss = css`
  border-radius: 50%;
  position: relative;
  top: -3px;
`;

export default UserProfileImg;
