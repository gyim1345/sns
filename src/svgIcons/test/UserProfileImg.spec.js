import React from 'react';
import { mount } from 'enzyme';
import UserProfileImg from '../UserProfileImg';

describe('<UserProfileImg />', () => {
  it('renders <img>', () => {
    const wrapper = mount(<UserProfileImg />);

    expect(wrapper.html()).toMatch('<img');
    expect(wrapper).toMatchSnapshot();
  });
})

