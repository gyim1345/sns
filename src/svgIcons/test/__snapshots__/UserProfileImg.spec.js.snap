// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserProfileImg /> renders <img> 1`] = `
<UserProfileImg>
  <EmotionCssPropInternal
    __EMOTION_LABEL_PLEASE_DO_NOT_USE__="UserProfileImg"
    __EMOTION_TYPE_PLEASE_DO_NOT_USE__="img"
    alt=""
    css={
      Object {
        "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlVzZXJQcm9maWxlSW1nLmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQU9rQiIsImZpbGUiOiJVc2VyUHJvZmlsZUltZy5qcyIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG5pbXBvcnQgeyBjc3MgfSBmcm9tICdAZW1vdGlvbi9jb3JlJztcblxuZnVuY3Rpb24gVXNlclByb2ZpbGVJbWcoeyB1c2VySW1hZ2UgfSkge1xuICByZXR1cm4gPGltZyBjc3M9e1tpbWdDc3NdfSBzcmM9e3VzZXJJbWFnZX0gYWx0PVwiXCIgd2lkdGg9ezI4fSBoZWlnaHQ9ezI4fSAvPjtcbn1cblxuY29uc3QgaW1nQ3NzID0gY3NzYFxuICBib3JkZXItcmFkaXVzOiA1MCU7XG4gIHBvc2l0aW9uOiByZWxhdGl2ZTtcbiAgdG9wOiAtM3B4O1xuYDtcblxuZXhwb3J0IGRlZmF1bHQgVXNlclByb2ZpbGVJbWc7XG4iXX0= */",
        "name": "purn1n-imgCss-UserProfileImg",
        "next": undefined,
        "styles": "border-radius:50%;position:relative;top:-3px;;label:imgCss;;;label:UserProfileImg;/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlVzZXJQcm9maWxlSW1nLmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUljIiwiZmlsZSI6IlVzZXJQcm9maWxlSW1nLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0JztcbmltcG9ydCB7IGNzcyB9IGZyb20gJ0BlbW90aW9uL2NvcmUnO1xuXG5mdW5jdGlvbiBVc2VyUHJvZmlsZUltZyh7IHVzZXJJbWFnZSB9KSB7XG4gIHJldHVybiA8aW1nIGNzcz17W2ltZ0Nzc119IHNyYz17dXNlckltYWdlfSBhbHQ9XCJcIiB3aWR0aD17Mjh9IGhlaWdodD17Mjh9IC8+O1xufVxuXG5jb25zdCBpbWdDc3MgPSBjc3NgXG4gIGJvcmRlci1yYWRpdXM6IDUwJTtcbiAgcG9zaXRpb246IHJlbGF0aXZlO1xuICB0b3A6IC0zcHg7XG5gO1xuXG5leHBvcnQgZGVmYXVsdCBVc2VyUHJvZmlsZUltZztcbiJdfQ== */",
        "toString": [Function],
      }
    }
    height={28}
    width={28}
  >
    <img
      alt=""
      className="css-purn1n-imgCss-UserProfileImg"
      height={28}
      width={28}
    />
  </EmotionCssPropInternal>
</UserProfileImg>
`;
